README.md

```diff
@@ -8,14 +8,17 @@ contexts too.
 Installation
 ------------
 
-Once you have [installed Go][golang-install], install the `mockgen` tool:
+Once you have [installed Go][golang-install], install the `mockgen` tool.
+
+To get the latest released version use:
 
 ```bash
-go get github.com/golang/mock/mockgen
+GO111MODULE=on go get github.com/golang/mock/mockgen@latest
```

Comment:
As a part of our next release I think we should start uploading artifacts as well. This will allow more people to get the desired behavior out of version. We can use a recent contribution to help with this: https://github.com/golang/mock/pull/229

Reply:
Cool! Yes if we build binaries ourselves and we build it with 1.12+ version of golang, then the built binaries should support the -version flag as implemented here.